addons/mass_mailing/models/mailing_list.py

```diff
@@ -126,6 +126,33 @@ def copy(self, default=None):
     # ACTIONS
     # ------------------------------------------------------
 
+    def action_open_import(self):
+        """Open the mailing list contact import wizard."""
+        self.ensure_one()
+        action = self.env['ir.actions.actions']._for_xml_id('mass_mailing.mailing_contact_import_action')
+        action['context'] = {
+            'default_mailing_list_ids': self.ids,
+            'default_subscription_list_ids': [(0, 0, {'list_id': self.id})],
```

Comment:
Could we support "multi" here by issuing multiple o2m commands?

Reply:
you want me to use Command.create ?
(not sure to understand sorry)

Reply:
Sorry it was a bit unclear, the idea would be to support multi-records in this method (in other words removing the ensure_one).

Would maybe be possible by building o2m commands like so:
`'default_subscription_list_ids': [(0, 0, {'list_id': list.id}) for list in self],`

Do you see what I mean?

Reply:
Yes, indeed it's better :)

I also removed the ensure_one